src/endpoint/iam/ops/iam_list_user_tags.js

```diff
@@ -14,20 +14,19 @@ async function list_user_tags(req, res) {
     const params = {
         username: req.body.user_name,
         marker: req.body.marker,
-        max_items: iam_utils.parse_max_items(req.body.max_items) ?? iam_constants.DEFAULT_MAX_ITEMS,
+        max_items: iam_utils.parse_max_items(req.body.max_items) ?? iam_constants.DEFAULT_MAX_TAGS,
```

Comment:
There no need to change, according to the AWS docs ([link](https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListUserTags.html)):

> If you do not include this parameter, the number of items defaults to 100.